analysis_driver/dataset.py

```diff
@@ -458,6 +459,14 @@ def run_elements(self):
             )
         return self._run_elements
 
+    @property
+    def sample(self):
+        if self._sample is None:
+            self._sample = rest_communication.get_document(
+                'aggregate/samples', match={'sample_id': self.name}
```

Comment:
We should use the new aggregation now in the standard endpoints, e.g. `get_document('samples', where={'sample_id': self.name})['aggregated']`